rest/config.go

```diff
@@ -943,15 +939,17 @@ func ParseCommandLine() (err error) {
 	return err
 }
 
-func SetMaxFileDescriptors(maxP *uint64) {
+func SetMaxFileDescriptors(maxP *uint64) error {
```

Comment:
I think we should update where this is being called to log the warning on error. It prevents an unhandled error, and prevents additional warnings from being logged inside tests that only care about the returned error.

Reply:
This has been fixed and changes are available in the latest revision.